man/sg.1.xml

```diff
@@ -73,6 +75,39 @@
     </para>
   </refsect1>
 
+  <refsect1 id='options'>
+    <title>OPTIONS</title>
+    <para>
+      The options which apply to the <command>sg</command> command are:
+    </para>
+    <variablelist remap='IP'>
+      <varlistentry>
+	<term><option>-</option>, <option>-l</option></term>
+	<listitem>
+	  <para>
+	    Provide an environment
+	    similar to what the user would expect
+	    had the calling user logged in directly.
```

Comment:
About `logged in directly`, I'm not sure.  What would `directly` mean here?

How about something simpler?:

```
Start the shell as a login shell.
```